fractal_server/app/api/v1/workflow.py

```diff
@@ -125,6 +128,38 @@ async def _get_workflow_task_check_owner(
     return workflow_task, workflow
 
 
+async def _check_new_workflow_name_and_project_id(
```

Comment:
change name from `_check_new_workflow_name_and_project_id` to `_check_workflow_exists`